frontend/app/components/input-textarea.tsx

```diff
@@ -0,0 +1,84 @@
+import { InputError } from '~/components/input-error';
+import { InputHelp } from '~/components/input-help';
+import { InputLabel } from '~/components/input-label';
+import { cn } from '~/utils/tailwind-utils';
+
+const inputBaseClassName =
+  'block rounded-lg border-gray-500 focus:border-blue-500 focus:outline-none focus:ring focus:ring-blue-500';
+const inputDisabledClassName =
+  'disabled:bg-gray-100 disabled:pointer-events-none disabled:cursor-not-allowed disabled:opacity-70';
+const inputReadOnlyClassName =
+  'read-only:bg-gray-100 read-only:pointer-events-none read-only:cursor-not-allowed read-only:opacity-70';
+const inputErrorClassName = 'border-red-500 focus:border-red-500 focus:ring-red-500';
+
+export interface InputTextareaProps
+  extends OmitStrict<
+    React.ComponentProps<'textarea'>,
+    'aria-describedby' | 'aria-errormessage' | 'aria-invalid' | 'aria-labelledby' | 'aria-required'
+  > {
+  errorMessage?: string;
+  helpMessage?: React.ReactNode;
+  id: string;
+  label: string;
+  name: string;
+}
+
+export function InputTextArea({
+  errorMessage,
+  className,
+  helpMessage,
+  id,
+  label,
+  required,
+  rows,
+  ...restInputProps
+}: InputTextareaProps) {
+  const inputErrorId = `input-${id}-error`;
+  const inputHelpMessageId = `input-${id}-help`;
+  const inputLabelId = `input-${id}-label`;
+  const inputWrapperId = `input-${id}`;
+
+  function getAriaDescribedby() {
+    if (helpMessage) {
+      return inputHelpMessageId;
+    }
+  }
+
+  return (
+    <div id={inputWrapperId} data-testid={inputWrapperId} className="form-group">
+      <InputLabel id={inputLabelId} htmlFor={id} className="mb-2">
+        {label}
+      </InputLabel>
+      {errorMessage && (
+        <div className="mb-2">
+          <InputError id={inputErrorId}>{errorMessage}</InputError>
+        </div>
+      )}
+      <textarea
+        aria-describedby={getAriaDescribedby()}
```

Comment:
Sorry.. just realized.. can this be just `aria-describedby={helpMessage && inputHelpMessageId}`?